app/controllers/orders_controller.rb

```diff
@@ -94,4 +95,12 @@ def resource_set(resource = nil)
     resource ||= Order.find_by!(code: params[:id])
     instance_variable_set("@#{resource_name}", resource)
   end
+
+  def homs_config
+    Rails.application.config.app
+  end
+
+  def use_imprint?
+    homs_config[:use_imprint] && @order.print_form_code.present?
```

Comment:
Don't need ```@order.print_form_code.present?```

It's better to make "print" button inactive when ```print_form_code``` is not present and show message "For printing set print form code for order type <order_type_code>".